RFC-001-BAM.md

```diff
@@ -119,15 +125,64 @@ The messages are also not dynamic as their structure needs to be known at compil
 
 Given that, we decided against gRPC.
 
-## Introducing **BAM!**
+### libp2p
 
-Given the above evaluation, we are left with defining our own protocol that implements the stated requirements.
-It incorporates ideas from several other protocols and combines them for our usecase.
-In particular, it borrows ideas from the following protocols:
+Libp2p is a modular network stack extracted from the ipfs (interplanetary file system) project.
+It provides reusable modules for common needs of peer-to-peer networking applications like discovery mechanisms, abstractions of various transport protocols, pluggable transport layer encryption and multiplexing.
 
-- BEEP
-- HTTP2
-- Lightning Network
+These are all features we find want for the transport layer of the COMIT network and hence we decided to build the communication layer on top of libp2p.
+
+## Identification & configuration
+
+Application protocols in libp2p have an identifier which allows nodes to exchange, which protocols they understand.
+For COMIT and the messages described in this document, we use the protocol identifier: `/comit/1.0.0`.
+
+The libp2p stack of COMIT is configured as follows:
+
+- tcp as the transport layer
+- yamux as the multiplexing module
```

Comment:
```suggestion
- Yamux as the multiplexing module
```

Reply:
hmm, this change may make `secio` below seem odd.  I'm not sure whether it should be `Secio` or not?  I couldn't find typical usage on the web.